src/components/sbb-button/sbb-button.tsx

```diff
@@ -151,6 +152,20 @@ export class SbbButton implements LinkButtonProperties, ComponentInterface {
     }
   }
 
+  @Listen('click')
+  public handleClick(event: Event): void {
+    if (this.disabled) {
```

Comment:
I wonder if we should provide an additional function `preventOrForwardHostClick` which additionally checks the disabled state?

Reply:
I'd leave it as it is, as then the disabled state becomes a little bit hidden. Maybe I have to explain it in a call ;)